### PR TITLE
README cleanup

### PR DESCRIPTION
**Pull request by ayanonagon**
Noticed that “CocoaPods” was spelled incorrectly and figured we should fix it before @neonichu or @kylef see this. :wink: 

Also fixed some other stuff while at it! Hoping to be able to contribute something to the Swift code soon as well. :octocat:

### CODE REVIEW

**Comment by neonichu** [comment]
@ayanonagon :+1:

File: README.md [hunk 1]
```diff
@@ -49,110 +49,131 @@ To install CryptoSwift, add it as a submodule to your project (on the top level
...
+```swift
+// Calculate Message Authentication Code (MAC) for message
+let mac = Authenticator.Poly1305(key: key).authenticate(message)
+```
+
 ##Contact
 Marcin Krzyżanowski [@krzyzanowskim](http://twitter.com/krzyzanowskim)
 
-##Licence
+##License
```
Comment (kylef) [inline README.md]: :gb: :+1:

**Comment by kylef** [comment]
:balloon:

File: README.md (hunk 1 above)
Comment (ayanonagon) [inline README.md]: Ugh TIL.  Fine I’ll switch that back. :grimacing:

**Comment by krzyzanowskim** [comment]
this is getting serious: http://www.grammar-monster.com/easily_confused/licence_license.htm ;-)

**Pull request merged**